src/Asset/TagRenderer.php

```diff
@@ -18,9 +18,12 @@ final class TagRenderer
 
     private $packages;
 
+    private $crossorigin;
+
     public function __construct(
         $entrypointLookupCollection,
-        Packages $packages
+        Packages $packages,
+        string $crossorigin
```

Comment:
Maybe it would be better to have a `$defaultAttributes` array here instead? in case we want to be able to configure more of them later?

Reply:
Yes. Done.